### DISCUSSION TITLE
Difference between stylex.props and stylex()

### DISCUSSION THREAD

**Discussion by jolbol1** [facebook/stylex#85] [Q&A] [answered]
I noticed in the NextJS demo that this method is used to apply styles
` <main className={stylex(s.main)}>`

I havent seen this in the documentation, which always suggests this method
`<body {...stylex.props(styles.reset, styles.body)}>{children}</body>`

Is there a difference?

**Comment by necolas** [comment, marked as answer]
`stylex()` is deprecated and should not be used - we just haven't quite finished transitioning off it internally
